### ISSUE TITLE
[FR] Searching/Ordering in referenced collections/fields

### ISSUE THREAD

**Issue by Bassadin** [mingo-app/mingo#649]
**Is your feature request related to a problem? Please describe.**
![image](https://github.com/user-attachments/assets/8f2d7585-e2be-4e8a-a8d8-5a12cbcee8ef)
I'd love to be able to search for referenced collections here and/or have some sort of ordering for them. We have collections where objects can have up to 30 referencing collection and it gets cluttered really fast.

**Describe the solution you'd like**
A search field or maybe alphabetic ordering at the very least :)

**Describe alternatives you've considered**
Searching manually :D Takes time.

**Comment by tothradoslav** [comment]
done, there will be a filter input, autofocused, when this dropdown / context menu is opened. Just start typing and will filter the options.

**Comment by Bassadin** [comment]
![image](https://github.com/user-attachments/assets/c56073ce-3217-4257-92fc-9cc4bbef491c)

Thank you so much! :D

**Comment by Bassadin** [comment]
@tothradoslav I don't think this is available in 2.0.1 yet. Any idea which future version this will appear in? :) Or do I need to enable it somewhere?

**Comment by Bassadin** [comment]
@tothradoslav will this be in v2025.11.0? :D

**Comment by tothradoslav** [comment]
Sorry for the late reply. Yes this will be available in next release.

**Comment by Bassadin** [comment]
Very cool! :)

**Comment by tothradoslav** [comment]
This has been fixed in release candidate [v2025.11.0-rc3](https://github.com/mingo-app/mingo/releases/tag/v2025.11.0-rc3).

Please test and verify the fix. This will be included in the next production release.

If you encounter any issues with the fix or implementation, please reply to this issue.

**Comment by Bassadin** [comment]
Hey, thanks for implementing it. It kinda works like I expected, however, it would be nice, if I could just down/up-arrow my way to the relation items and press enter to direkt jump into the corresponding collections. Would you consider that part of this FR or should I open a new one?

**Comment by lschoeb** [comment]
Also, it would be great if the new tab continues to open just to the right of the current one, it it used to.
In the rc3, it opens left of the home tab all the way on the left. This is confusing for tab management.

**Comment by tothradoslav** [comment]
I will create a separate task for the latter 2 FRs. Thanks!

**Comment by tothradoslav** [comment]
This has been fixed and released in version [2025.12.0](https://github.com/mingo-app/mingo/releases/tag/v2025.12.0).

Thank you for reporting this issue!

**Comment by lschoeb** [comment]
I can see #740 for my FR above, which is resolved in the release candidate.
But I cannot find an issue for @Bassadin s FR (down/up-arrow navigation for search results)
@tothradoslav

**Comment by tothradoslav** [comment]
This has been implemented globally in our context menu. You may now use up / down keys for navigating options. Use right to move into submenu, left to move back to main menu. Enter to chose.

**Comment by tothradoslav** [comment]
This has been fixed and released in version [2026.2.0](https://github.com/mingo-app/mingo/releases/tag/v2026.2.0).

Thank you for reporting this issue!
